# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/pcmcia-cs/xcardinfo-pcmcia-cs-include.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./cardmgr/xcardinfo.c.orig	2005-01-26 15:53:25.968206912 +0100
+++ ./cardmgr/xcardinfo.c	2005-01-26 15:53:54.293900752 +0100
@@ -44,7 +44,7 @@
 #include <pcmcia/config.h>
 #include <pcmcia/version.h>
 #include <pcmcia/cs_types.h>
-#include <pcmcia/cs.h>
+#include "../include/pcmcia/cs.h"
 #include <pcmcia/cistpl.h>
 #include <pcmcia/ds.h>
 
